core/lib/spree/money.rb

```diff
@@ -6,11 +6,22 @@ module Spree
   # Spree::Money is a relatively thin wrapper around Monetize which handles
   # formatting via Spree::Config.
   class Money
+    RUBY_NUMERIC_STRING = /\A-?\d+(\.\d+)?\z/
+
     class <<self
       attr_accessor :default_formatting_rules
 
       def from_money(money)
-        new(money.to_d, currency: money.currency.iso_code)
+        new(money)
```

Comment:
This is becoming the ultimate bikeshedding exercise, but: This method is now useless. Can you please please get rid of it?